onyo/lib/onyo.py

```diff
@@ -95,6 +152,11 @@ def clear_caches(self, assets: bool = True, dirs: bool = True,
 
         By default all caches are cleared, and flags make it possible to specify
         which caches which should remain set.
+
+        If the repository is only modified via the API functions, the caches of
+        the `Repo` object are kept consistent. If an external process modifies
```

Comment:
nit: remove external. Internally, it's /still/ the responsibility to correctly manage the caches.